pkg/entities/types.go

```diff
@@ -16023,6 +16040,9 @@ type entitySearchResponse struct {
 // AgentApplicationSettingsErrorCollectorHttpStatus - A list of HTTP status codes and/or status code ranges, such as "404" or "500-599"
 type AgentApplicationSettingsErrorCollectorHttpStatus string
 
+// AgentApplicationSettingsRawJsConfiguration - The "raw" configuration values for configuring the javascript client.
+type AgentApplicationSettingsRawJsConfiguration map[string]interface{}
```

Comment:
The type of this attribute was generated as a `string` by Tutone, which is accurate, given the attribute on NerdGraph (`jsConfig`) has no nested attributes. However, I've manually edited this to a `map[string]interface{}` as the calling function seems to read this attribute as a `map[string]interface{}`, and keeping it as a string seems throws an error when `terraform apply` is run with the Terraform changes, stating a map[string]interface{} was found where a string was expected.